conjur/policy/policy_controller.py

```diff
@@ -23,5 +28,11 @@ def load(self):
         """
         Method that facilitates all method calls in this class
         """
-        result = self.policy_logic.run_action(self.policy_data)
-        sys.stdout.write(result+'\n')
+        try:
+            result = self.policy_logic.run_action(self.policy_data)
+            sys.stdout.write(result+'\n')
+        except requests.exceptions.HTTPError as http_error:
+            if http_error.response.status_code == 422:
+                raise InvalidFormatException(f"{http_error}. The policy was empty or its contents "
+                                             "were not in valid YAML.") from http_error
```

Comment:
How about - or its content was not a valid YMAL.

Reply:
I think its better' or its content was not in valid YAML' or 'not in valid YAML syntax'?

Reply:
We need Shuli approval here.. how do you want to handle it?
BTW, how come Shuli is not a user in this repo?

Reply:
I will send Shuli a message. Don't know why

Reply:
Do I have the power to add her?

Reply:
@sigalsax  - approved.
Weird that you can't tag me.